src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/DefendantResponseNotificationService.java

```diff
@@ -130,7 +130,9 @@ public void logNotificationFailure(
     }
 
     private Map<String, String> aggregateParams(final Claim claim, final ResponseData responseData) {
-        boolean isFreeMediationRequested = responseData.getFreeMediation().equals(ResponseData.FreeMediationOption.YES);
+        boolean isFreeMediationRequested = responseData.getFreeMediation()
```

Comment:
What happens when `isFreeMediationRequested` to NO? I wonder whether we should be cautious as in the scenario below.